pyinstaller/specterd.spec

```
@@ -14,7 +14,10 @@ elif platform.system() == 'Linux':
     if platform.processor() == 'aarch64': #ARM 64 bit
         binaries = [("/lib/aarch64-linux-gnu/libusb-1.0.so.0", ".")]
     else:
-        binaries = [("/lib/x86_64-linux-gnu/libusb-1.0.so.0", ".")]
+        if distro.name() == "Arch Linux":
+            binaries = [("/usr/lib/libusb-1.0.so.0", ".")]
+        else:
+            binaries = [("/lib/x86_64-linux-gnu/libusb-1.0.so.0", ".")]
```

I suggest doing it slightly differently:
to set up a list of candidates where the library can live in different linux distros, and try to find one:
```suggestion
        candidates = [
                "/usr/lib/libusb-1.0.so.0",
                "/lib/x86_64-linux-gnu/libusb-1.0.so.0",
                "/lib/aarch64-linux-gnu/libusb-1.0.so.0",
                "/lib/arm-linux-gnueabihf/libusb-1.0.so.0",
        ]
        binaries = []
        for p in candidates:
            if os.path.isfile(p):
                binaries = [(p, ".")]
                break
```